haskell/src/Indigo/Contracts/Safelist.hs

```diff
@@ -0,0 +1,70 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Indigo.Contracts.Safelist
```

Comment:
Please add a comment explaining what it is. AFAIU it's a simplest implementation of safelist interface that we use for testing.

Reply:
Added